packages/documentation/src/stories/packages/components-react/components-react.stories.ts

```diff
@@ -0,0 +1,16 @@
+import { StoryObj } from '@storybook/web-components';
+import { MetaExtended } from '@root/types';
+
+const meta: MetaExtended = {
+  id: '13b9c7f1-993d-4348-a3b7-a7ceb92fd5c7',
+  title: 'Packages/Components for React',
+  parameters: {
+    badges: [],
+  },
```

Comment:
This can be removed, this is legacy code because we had storybook badges in the past.
```suggestion
```